__init__.py

```diff
@@ -1350,7 +1352,10 @@ def _properties_from_csv_row(row, header, ignored_columns):
             else:
                 try:
                     # We use literal_eval() here to de-stringify numbers, lists and objects in the CSV data
-                    p = literal_eval(row[x])
+                    # p = literal_eval(row[x])
+                    p = json.loads(row[x])
+                    if isinstance(p, list):
+                        Mixpanel.LOGGER.debug(str(p))
```

Comment:
do we need this debug statement?

Reply:
removed in later a commit on the full PR